lib/ansible/modules/packaging/os/service_facts.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/python
+# (c) 2017, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# originally copied from AWX's scan_services module to bring this functionality
+# into Core
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+import re
+from ansible.module_utils.basic import * # noqa
+
+DOCUMENTATION = '''
+---
+module: scan_services
+short_description: Return service state information as fact data
+description:
+     - Return service state information as fact data for various service management utilities
+version_added: "1.9"
```

Comment:
`version_added: "2.5"`

Reply:
Ugh, that was a terrible oversight from the brought-over module from AWX. Thanks.